### DISCUSSION TITLE
Month in Revolt - July 2024 - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#765] [Announcements]
# Month in Revolt - July 2024 - Revolt

The monthly blog update for June–July period.

https://revolt.chat/posts/month-in-revolt-july-2024

**Comment by MCausc78** [comment]
real 🇺🇦🤯

**Comment by Andre601** [comment]
Not a big fan of the change from spoilers using vertical pipes... I found the exclamation ones to be a bit more accessible than the vertical pipe ones...

**Reply by ghost** [reply]
Only thing this changes is for ppl who are familiar with discord to be able to easily spoiler things

**Reply by skyH4PPY** [reply]
messages like this would be spoilered otherwise: hey!! this is bad!!
me and my friends had to deal with half our messages being spoilered when we first tried revolt so we much prefer the way discord handles it

**Comment by X-lem** [comment]
Love the change to using pipes instead of exclamation points for spoilers! It's more in line with Discord so people will have an easier time moving over. Also avoids accidental spoilers when people type something like `This is so cool!! Way to go!!`

All the MD changes look 🔥

**Comment by searinminecraft** [comment]
yet another massive W for Revolt, as always.

**Comment by FanMclaine** [comment]
What the sigma :fire:
